### ISSUE TITLE
Add text highlighting to preview

### ISSUE THREAD

**Issue by bbringhu** [AdobeDocs/adobe-markdown-authoring#3]
We allow writers to use `class="preview"` in HTML to highlight text in yellow.

https://experienceleague.adobe.com/docs/authoring-guide-exl/using/markdown/syntax-style-guide.html?lang=en#text-highlighting

**Comment by geverit4** [comment]
I do not have access to this document.  

https://experienceleague.adobe.com/docs/authoring-guide-exl/using/markdown/syntax-style-guide.html?lang=en#text-highlighting

**Comment by bbringhu** [comment]
Here are the instructions:

## Text highlighting

The Workfront team asked to be able to use yellow highlighting to indicate the preview of upcoming features. Here's how it works.

Example:

```
This entire paragraph should NOT be highlighted. <span class="preview"> This word is **bold** inside a highlighted sentence.</span> And this is just the last sentence.
```

Rendered:

This entire paragraph should NOT be highlighted. <span class="preview"> This word is **bold** inside a highlighted sentence.</span> And this is just the last sentence.

As a general rule, use `<span class="preview">` to highlight a paragraph or text within a paragraph, and use `<div class="preview">` for multiple paragraphs and components.

>[!NOTE]
>
>We're still working on improving the highlighting display of certain page elements such as notes and tables. Feel free to log JIRA bugs if you see improper rendering. In progress.
>
>VSC preview does not yet support highlighting.

**Comment by geverit4** [comment]
Text highlighting now works: 

![Screenshot 2023-04-27 at 5 48 27 PM](https://user-images.githubusercontent.com/1187616/234998723-d8ed049f-9df1-4fe3-adf7-f483e3e23a22.png)

**Comment by bbringhu** [comment]
Nice!

**Comment by bbringhu** [comment]
Confirmed fixed in 2.0.5
